newdle/client/package.json

```diff
@@ -51,7 +51,6 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "add-locale": "lingui add-locale",
```

Comment:
This command is now deprecated, it's enough to just add the locale in the lingui config